src/Zicht/Bundle/MenuBundle/Provider/DatabaseMenuProvider.php

```diff
@@ -50,7 +50,7 @@ public function __construct(BuilderInterface $builder, ContainerInterface $conta
      */
     public function get($name, array $options = array())
     {
-        return $this->builder->build($name, $this->container->get('request'));
+        return $this->builder->build($name, $this->container->get('request_stack')->getMasterRequest());
```

Comment:
to be sure, you want to send master request? because parameters added to subrequest will be lost?

Reply:
Changed to `getCurrentRequest`